webapp/src/js/enketo/medic-xpath-extensions.js

```diff
@@ -14,11 +14,54 @@ var getValue = function(resultObject) {
   return resultObject.v;
 };
 
+var now_and_today = function() { return { t: 'date', v: new Date() }; };
+
+Date.prototype.toISOLocalString = function() {
```

Comment:
Don't add to the prototype if you can avoid it. This could break if some library or browser decided to implement a function also called `toISOLocalString` but implemented it differently. In this case I think it's unnecessary as we can just call a private function from the `toExternalResult` function. That also reduces the scope from global to private which is always good if you can!